frontend/packages/console-dynamic-plugin-sdk/docs/api.md

```diff
@@ -2077,6 +2078,27 @@ Component that allows to receive contributions from other plugins for the `conso
 
 
 
+---
+
+## `NamespaceBar`
+
+### Summary 
+
+A component that renders a namespace dropdown menu.
+
+
+
+
+### Parameters
+
+| Parameter Name | Description |
+| -------------- | ----------- |
+| `isDisabled` | (optional) A boolean flag that disables the dropdown if set to true |
+| `onNamespaceChange` | (optional) A function that is executed when a namespace option is selected. The only argument it should accept is new namespace as a string. |
```

Comment:
It feels like this should be required unless the namespace bar itself handles the context update in Redux -- and if it does we should state that in the description along with how it affects the URL.

Reply:
The active namspace is updated in `NamespaceBarDropdowns`, which is used by `NamespaceBar`. I updated the description as suggested.